capstone-backend/src/main/java/com/google/sps/servlets/MeetingServlet.java

```diff
@@ -95,13 +95,17 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     if (keySet.contains("startDateTime")) {
       long startDateTime = jsonObject.get("startDateTime").getAsLong();
       DateTime dateTime = new DateTime(startDateTime);
-      EventDateTime start = new EventDateTime().setDateTime(dateTime);
+      EventDateTime start = new EventDateTime()
+                              .setDateTime(dateTime)
+                              .setTimeZone(jsonObject.get("timezone").getAsString());
       event.setStart(start);
     }
     if (keySet.contains("endDateTime")) {
       long endDateTime = jsonObject.get("endDateTime").getAsLong();
       DateTime dateTime = new DateTime(endDateTime);
-      EventDateTime end = new EventDateTime().setDateTime(dateTime);
+      EventDateTime end = new EventDateTime()
+                          .setDateTime(dateTime)
+                         .setTimeZone(jsonObject.get("timezone").getAsString());
```

Comment:
nit: can you update the formatting of the these lines?

Reply:
Sure!